src/components/event-listing/EventListing.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Card, Divider, Button } from 'antd';
+
+// placeholder interface for now (will replace with real interface once hooked up to db)
+interface Event {
+  title: string;
+  date: string;
+  time: string;
+  description: string;
+  otherNotes?: string;
+  // TODO: use correct file type
+  thumbnail?: string;
+}
+
+const StyledCard = styled(Card)`
+  margin-bottom: 12px;
+`; 
+
+const CardContent = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const H1 = styled.h1`
```

Comment:
I think for these H1 and H3 tags it's probably best to use Typography from Ant.design. https://3x.ant.design/components/typography/